tests/kafkatest/tests/streams/streams_broker_bounce_test.py

```diff
@@ -217,11 +209,6 @@ def collect_results(self, sleep_time_secs):
             num_threads=[1, 3],
             sleep_time_secs=[120],
             metadata_quorum=[quorum.isolated_kraft])
-    @matrix(failure_mode=["clean_shutdown", "hard_shutdown", "clean_bounce", "hard_bounce"],
-            broker_type=["leader", "controller"],
-            num_threads=[1, 3],
-            sleep_time_secs=[120],
-            metadata_quorum=[quorum.zk])
```

Comment:
Why are we removing this matrix?

Reply:
That matrix is using zookeeper

Reply:
Ah. I missed the other `@matrix` annotation above it.

Seems we can remove `metadata_quorum=[quorum.isolated_kraft]` from the remaining matrix?